### DISCUSSION TITLE
Make this plugin for Davinci Resolve!

### DISCUSSION THREAD

**Discussion by The3IC** [intel/openvino-plugins-ai-audacity#192] [Ideas]
Not sure where to really post this but the features (and in particular the transcribing and translating) would be just so useful if they were native in Davinci Resolve, so pls OpenVINO, port the stuff to Resolve!

Here you go: https://gist.github.com/X-Raym/2f2bf453fc481b9cca624d7ca0e19de8

**Comment by MarcoRavich** [comment]
...[JUCE](https://juce.com/juce/features/) - aka VST3/AU3/LV2/etc - is [the way to go](https://github.com/intel/openvino-plugins-ai-audacity/discussions/77) then !
